frontend/src/components/SideBar.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+
+  import Button from "./Button.svelte";
+
+   export let show_menu = false;
+   export let show_legend = false;
+   export let show_timebar = false;
+   export let updateTimeBarPlaceholder:any; //it was called directly inside the div element, now: a method is created inside app.svelt
+   export let toggleMenuPlaceholder: any;  
+   export let toggleLegendPlaceholder: any;
```

Comment:
Types needs to be defined.

Reply:
Since the declared methods return Objects, what do you suggest the data type be? 
It can be `Object` but I thought any was better than `Object.` 
ex: 
`export let toggleLegendPlaceholder: Object; `